MastodonSDK/Sources/MastodonCore/Service/InstanceService.swift

```diff
@@ -34,90 +34,64 @@ public final class InstanceService {
             .receive(on: DispatchQueue.main)
             .compactMap { $0.first?.domain }
             .removeDuplicates()     // prevent infinity loop
-            .sink { [weak self] domain in
-                guard let self = self else { return }
-                self.updateInstance(domain: domain)
+            .asyncMap { [weak self] in await self?.updateInstance(domain: $0) }
+            .sink { _ in
+                // no-op
```

Comment:
Why is this still needed?

Reply:
Without a `sink` the pipeline is not observer and no events will be emitted.